rocksdb.nimble

```diff
@@ -1,16 +1,14 @@
-packageName   = "rocksdb"
```

Comment:
I presume this is `nph` - should add a github action to maintain

Reply:
(and it should really have gone into a separate pr)

Reply:
> I presume this is `nph` - should add a github action to maintain

Yes I had nph enabled in vscode when saving files. Do you mean a nph check step to fail CI when nph hasn't been run on the files?

Reply:
> (and it should really have gone into a separate pr but whatever, let's just add the github action enforcer)

Yes good point, thanks for the reminder. I'll try to do that way in the future.

Reply:
> Do you mean a nph check step to fail CI when nph hasn't been run on the files?

yes, without that we'll be seeing back-and-forths

Reply:
one idea is to make the nph change and rebase this pr on top - usually, this works pretty well since git can usually figure it out when two branches make the same changes

Reply:
Ok, I'll try that.